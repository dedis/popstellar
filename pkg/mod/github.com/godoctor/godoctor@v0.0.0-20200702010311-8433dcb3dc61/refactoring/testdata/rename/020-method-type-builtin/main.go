package main

import "fmt"


type simple string

// Test for renaming method in interface

func main() {

var simplevar simple = "hellooo"


simplevar.mymethod()		// <<<<< rename,15,12,15,12,renamed,pass


}

func (simplevar simple)mymethod() {


fmt.Println(simplevar)


}


