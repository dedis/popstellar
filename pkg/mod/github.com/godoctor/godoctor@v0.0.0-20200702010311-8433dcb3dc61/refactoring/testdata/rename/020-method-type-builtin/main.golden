package main

import "fmt"


type simple string

// Test for renaming method in interface

func main() {

var simplevar simple = "hellooo"


simplevar.renamed()		// <<<<< rename,15,12,15,12,renamed,pass


}

func (simplevar simple)renamed() {


fmt.Println(simplevar)


}


